components/brave_ads/common/custom_notification_ads_features.cc

```diff
@@ -94,128 +73,107 @@ constexpr int kDefaultNotificationAdInsetX = -(10 + kNativeNotificationWidth);
+    kDefaultNotificationAdInsetX};
 
-// Ad notification normalized display coordinate for the y component should be
-// between 0.0 and 1.0; coordinates outside this range will be adjusted to fit
-// the work area. Set to 0.0 for top, 0.5 for middle or 1.0 for bottom
-constexpr char
-    kFieldTrialParameterNotificationAdNormalizedDisplayCoordinateY[] =
-        "ad_notification_normalized_display_coordinate_y";
 #if BUILDFLAG(IS_WIN)
 constexpr double kDefaultNotificationAdNormalizedDisplayCoordinateY = 1.0;
 #elif BUILDFLAG(IS_MAC)
 constexpr double kDefaultNotificationAdNormalizedDisplayCoordinateY = 0.0;
 #elif BUILDFLAG(IS_LINUX)
 constexpr double kDefaultNotificationAdNormalizedDisplayCoordinateY = 0.0;
 #endif
+// Ad notification normalized display coordinate for the y component should be
+// between 0.0 and 1.0; coordinates outside this range will be adjusted to fit
+// the work area. Set to 0.0 for top, 0.5 for middle or 1.0 for bottom
+constexpr base::FeatureParam<double>
+    kNotificationAdNormalizedDisplayCoordinateY{
+        &kCustomNotificationAds,
+        "ad_notification_normalized_display_coordinate_y",
+        kDefaultNotificationAdNormalizedDisplayCoordinateY};
 
-// Ad notification y inset within the display's work area specified in screen
-// coordinates
-constexpr char kFieldTrialParameterNotificationAdInsetY[] =
-    "ad_notification_inset_y";
 #if BUILDFLAG(IS_WIN)
 constexpr int kDefaultNotificationAdInsetY = -10;
 #elif BUILDFLAG(IS_MAC)
 constexpr int kDefaultNotificationAdInsetY = 11;
 #elif BUILDFLAG(IS_LINUX)
 constexpr int kDefaultNotificationAdInsetY = 18;
 #endif
+// Ad notification y inset within the display's work area specified in screen
+// coordinates
+constexpr base::FeatureParam<int> kNotificationAdInsetY{
+    &kCustomNotificationAds, "ad_notification_inset_y",
+    kDefaultNotificationAdInsetY};
 
 #endif  // !BUILDFLAG(IS_ANDROID) && !BUILDFLAG(IS_IOS)
 
 }  // namespace
 
-bool IsNotificationAdsEnabled() {
-  return base::FeatureList::IsEnabled(kNotificationAds);
-}
+BASE_FEATURE(kCustomNotificationAds,
+             "CustomAdNotifications",
+             base::FEATURE_DISABLED_BY_DEFAULT);
 
-bool CanFallbackToCustomNotificationAds() {
-  return GetFieldTrialParamByFeatureAsBool(
-      kNotificationAds, kFieldTrialParameterCanFallbackToCustomNotificationAds,
-      kDefaultCanFallbackToCustomNotificationAds);
+BASE_FEATURE(kAllowedToFallbackToCustomNotificationAds,
+             "AllowedToFallbackToCustomAdNotifications",
+             base::FEATURE_ENABLED_BY_DEFAULT);
+
+bool IsCustomNotificationAdsEnabled() {
+  return base::FeatureList::IsEnabled(kCustomNotificationAds);
 }
 
-int NotificationAdTimeout() {
-  return GetFieldTrialParamByFeatureAsInt(
-      kNotificationAds, kFieldTrialParameterNotificationAdTimeout,
-      kDefaultNotificationAdTimeout);
+bool CanFallbackToCustomNotificationAds() {
+  return kCanFallbackToCustomNotificationAds.Get();
```

Comment:
See other comment but ideally we only need Is*Enabled for convenience but all other k*.get calls can just be used